canonical/jaas-dashboard
# Notify controller juju version updates

## Done
- Show the count of controllers with an old Juju version in the sidenav
- Show an update available badge per controller in the controllers view and link it to the upgrade docs

## QA

- Pull code
- Run `dotrun clean && dotrun serve`
- Open http://0.0.0.0:8036/
- Make sure that the count is shows in the navbar (A way to have the badge if you don't have outdated controller, is by adding `!` at line 391 in juju/api)
- Make sure that the update badge is shown for controllers with version `< 3.0.*`

## Details

https://warthogs.atlassian.net/browse/WD-2390

## Screenshots

![image](https://user-images.githubusercontent.com/36013798/224311688-58712a28-2505-489f-8ca9-049d414c7570.png)
